controller/invitation.go

```diff
@@ -73,3 +73,28 @@ func (c *InvitationController) CreateInvite(ctx *app.CreateInviteInvitationConte
 
 	return ctx.Created()
 }
+
+func (c *InvitationController) AcceptInvite(ctx *app.AcceptInviteInvitationContext) error {
+	currentIdentity, err := login.LoadContextIdentityIfNotDeprovisioned(ctx, c.app)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
+	}
+
+	acceptCode, err := uuid.FromString(ctx.AcceptCode)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
+		}, "failed to accept invitation, invalid code")
+
+		return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterErrorFromString("AcceptCode", ctx.AcceptCode, "accept code is not a valid uuid"))
+	}
+
+	c.app.InvitationService().Accept(ctx, currentIdentity.ID, acceptCode)
+
+	log.Debug(ctx, map[string]interface{}{
+		"accepting-user-id": *currentIdentity,
+		"accept-code":       ctx.AcceptCode,
+	}, "invitation accepted")
+
+	return ctx.TemporaryRedirect()
```

Comment:
Set the `Location` header for the redirect? Add the redirect url to configuration, and also set it in the configmap https://github.com/fabric8-services/fabric8-auth/blob/master/openshift/auth.config.yaml#L53 . We could keep the redirect url as `/_home` for now. Could you please assign an issue to @rohitkrai03 and work with UX to get the page designed?

fyi: This is what we use for the email verification redirect https://console.rh-idev.openshift.com/console/project/dsaas-preview/browse/config-maps/auth

Reply:
I've introduced a new configuration parameter, `invitation.accepted.url` which can be used to configure where the user will be redirected to after accepting an invitation.

Reply:
Thanks, looks good !
> Add the redirect url to configuration, and also set it in the configmap https://github.com/fabric8-services/fabric8-auth/blob/master/openshift/auth.config.yaml#L53 

 Could you please also update the  openshift templates? I've opened a housekeeping issue for this  : https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2114 . Please ensure we merge that, before the auth PR :) 

Btw, for a separate discussion: https://github.com/fabric8-services/fabric8-auth/issues/546

Reply:
ignore: https://github.com/fabric8-services/fabric8-auth/pull/547/files